### PR TITLE
Fix #485 added projex.upgrade_tree

### LINKED ISSUE #485: Upgrade path for projex
- projex should mock pykern.pksetup.setup to get the values.
- strip classifiers
- run a new projex on the old values (description, name) and diff files
    - strip lines that can be ignored for each diff. if nothing left, copy file
    - setup.py will be removed so not compared

### PR DESCRIPTION
**Pull request by robnagler**
- Updated URLs for licenses, homepage, docs to https

### CODE REVIEW

File: pykern/pkcli/projex.py [hunk 1]
```diff
@@ -61,6 +62,34 @@
...
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python",
+        "Topic :: Utilities",
+    ]
+    + [c[1] for c in LICENSES.values()],
+)
+
 
 def init_rs_tree(description):
     """Initialize defaults for RadiaSoft project tree and call `init_tree`
```
Comment (rorour) [inline pykern/pkcli/projex.py:95]: L99 needs update

**Review by rorour — CHANGES_REQUESTED** [review]

File: pykern/pkcli/projex.py [hunk 2]
```diff
@@ -79,11 +108,11 @@ def init_rs_tree(description):
         "pip@radiasoft.net",
         description,
         "apache2",
-        "https://github.com/radiasoft/" + name,
+        "https://git.radiasoft.org/" + name,
```
[the remark this answers is not in the document]
Comment (robnagler) [reply pykern/pkcli/projex.py:111]: Fixed

File: pykern/pkcli/projex.py (hunk 1 above)
Comment (robnagler) [reply pykern/pkcli/projex.py:98]: ac871f1e50b24ded75eefd3b3e7393281322ec6d

File: pykern/pkcli/projex.py [hunk 3]
```diff
@@ -141,6 +247,20 @@ def _license(name, which):
         )
 
 
+def _pksetup_kwargs():
```
[the remark this answers is not in the document]
Comment (robnagler) [reply pykern/pkcli/projex.py:262]: ac871f1e50b24ded75eefd3b3e7393281322ec6d

File: pykern/pkcli/projex.py [hunk 4]
```diff
@@ -129,6 +159,82 @@ def init_tree(name, author, author_email, description, license, url):
...
+        if kwargs.url.startswith("https://git.radiasoft"):
+            rv += [
+                "Update project url to:",
+                f"    {kwargs.url}",
+            ]
+        return rv
+
+    def _url(old_url, name):
+        if old_url == f"https://github.com/radiasoft/{name}":
+            return f"https://git.radiasoft.org/{name}"
```
[the remark this answers is not in the document]
Comment (robnagler) [reply pykern/pkcli/projex.py]: ac871f1e50b24ded75eefd3b3e7393281322ec6d

File: pykern/pkcli/projex.py [hunk 5]
```diff
@@ -28,31 +29,31 @@
...
         "License :: OSI Approved :: GNU General Public License v2 (GPLv2)",
     ),
     "gpl3": (
-        "http://www.gnu.org/licenses/gpl-3.0.txt",
+        "https://www.gnu.org/licenses/gpl-3.0.txt",
         "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
     ),
     "lgpl2": (
-        "http://www.gnu.org/licenses/lgpl-3.0.txt",
+        "https://www.gnu.org/licenses/lgpl-3.0.txt",
```
[the remark this answers is not in the document]
Comment (robnagler) [reply pykern/pkcli/projex.py]: Good eye. ac871f1e50b24ded75eefd3b3e7393281322ec6d

**Review by rorour — APPROVED** [review]

**Pull request merged**